### ISSUE TITLE
Settings not saved after startup

### ISSUE THREAD

**Issue by stano-vittorio** [MonitorControl/MonitorControl#1679]
### Before opening the issue, have you...?

- [X] Searched for existing issues
- [X] Looked through [the wiki](https://github.com/MonitorControl/MonitorControl/wiki)
- [X] Updated MonitorControl to the latest version (if applicable)

### Describe the bug

Hello everyone! I would like to note that the settings in the monitor --> advanced settings section do not remain saved after macOS starts up after restarting or shutdown. In particular, the value I set in the Max DDC is not saved. Is there anything that can be done?

### Steps to reproduce

1. Impostare parametri impostazioni avanzate.
2. Riavviare macOS.
3. Controllare impostazioni precedenti oppure regolare la luminosità con la tastiera e si vede che va oltre la soglia precedentemente impostata.

### Expected behavior

Le impostazioni avanzate dovrebbero rimanere salvate. In particolare, ho impostato un limite di luminosità massima di 80 e oltre quel valore non dovrebbe andare (la luminosità viene regolata con la tastiera).

### Environment Information (please complete the following information)

```markdown
- macOS version: 15.1 (24B83)
- Mac model: Mac mini M1
- MonitorControl version: 4.3.3 Build 7123
- Monitor(s): LG HDR WQHD
- Apple Silicon/M1 (yes or no): yes
```

**Comment by waydabber** [comment]
Sorry for not responding to this. I assume the issue was resolved then.

**Comment by stano-vittorio** [comment]
Thanks for the answer. After a few restarts I checked and the settings remain saved. A question, if I use the app on a macbook and I want the app to control only the external monitor and the internal one to control it directly from the physical buttons, is it possible to separate the two things?

**Comment by waydabber** [comment]
No, because if you want to control the external display using the keyboard, the app must assume control over the native keys thus it needs to control the internal display as well.
